azure-pipelines.yml

```diff
@@ -20,7 +20,7 @@ variables:
 stages:
 - stage: Build
   pool:
-    vmImage: windows-latest
+    vmImage: windows-2019
```

Comment:
Looks we can merge main branch to solve this conflict.

Reply:
Thanks. I have merged changes from main branch.